.expeditor/update_version.sh

```diff
@@ -1,8 +1,9 @@
-
 #!/bin/sh
+
 set -evx
 
 version=$(cat VERSION)
 
 # Update version for dobi
-sed -i -r "s/^(\\s*)VERSION: \".+\"/\\1VERSION: \"$version\"/" .expeditor/build.docker.yml
\ No newline at end of file
+sed -i -r "s/^(\\s*)VERSION: \".+\"/\\1VERSION: \"$version\"/" .expeditor/build.docker.yml
+sed -i.bak -r "s/^version: .+/version: $version/" charts/omnitruck/Chart.yaml
```

Comment:
I think the `.bak` should be removed. It will create a backup file which I believe will end up committed to the repo.

Reply:
it will or you add it to a ignore file